docker/bin/zookeeperReady.sh

```diff
@@ -27,40 +28,62 @@ if [[ "$OK" == "imok" ]]; then
   set +e
   nslookup $DOMAIN
   if [[ $? -eq 1 ]]; then
-
     set -e
     echo "There is no active ensemble, skipping readiness probe..."
     exit 0
-
   else
     set -e
     # An ensemble exists, check to see if this node is already a member.
     # Check to see if zookeeper service for this node is a participant
     set +e
-    ZKURL=$(zkConnectionString)
-    set -e
-    MYID=`cat $MYID_FILE`
-    ROLE=`java -Dlog4j.configuration=file:"$LOG4J_CONF" -jar /root/zu.jar get-role $ZKURL $MYID`
+    # Extract resource name and this members ordinal value from pod hostname
+    HOST=`hostname -s`
+    if [[ $HOST =~ (.*)-([0-9]+)$ ]]; then
+        NAME=${BASH_REMATCH[1]}
+        ORD=${BASH_REMATCH[2]}
+    else
+        echo Failed to parse name and ordinal of Pod
+        exit 1
+    fi
+    MYID=$((ORD+1))
+    ONDISK_CONFIG=false
+    if [ -f $MYID_FILE ]; then
+      EXISTING_ID="`cat $DATA_DIR/myid`"
+      if [[ "$EXISTING_ID" == "$MYID" && -f $STATIC_CONFIG ]]; then
+        # If Id is correct and configuration is present under `/data/conf`
+          ONDISK_CONFIG=true
+          DYN_CFG_FILE_LINE=`cat $STATIC_CONFIG|grep "dynamicConfigFile\="`
+          DYN_CFG_FILE=${DYN_CFG_FILE_LINE##dynamicConfigFile=}
+          SERVER_FOUND=`cat $DYN_CFG_FILE | grep "server.${MYID}=" | wc -l`
+          if [[ "$SERVER_FOUND" == "0" ]]; then
+            echo "Server not found in ensemble. Exiting ..."
+            exit 0
```

Comment:
@pbelgundi as we discussed in slack, this should be an exit code of `1` indicating that this node is _not ready_ since its not in the ensemble.

Reply:
Done. Changed to exit 1 and tested.